vita3k/modules/SceAvPlayer/SceAvPlayer.cpp

```diff
@@ -163,17 +159,102 @@ static Ptr<uint8_t> get_buffer(const PlayerPtr &player, MediaType media_type,
     return buffer;
 }
 
+#include <modules/module_parent.h>
```

Comment:
Is there reason why you didn't use the common create thread function?

Reply:
nvm. I just realized that this is the only way to create thread. Might be wise to refactor thread creation. It looks boring to write 10 lines of code to just create a simple thread. (possibly in other pr)

Reply:
yes, @bookmist  have said create another pr for improve code and fix last game video no playing, have already improved 7 8 game here, only akiba keep no works